docs/usage/getting-started/running.md

```diff
@@ -35,39 +35,36 @@ The `renovate` npm package is compatible with all of Renovate's supported platfo
 Renovate is also distributed as Docker images on Docker Hub (`renovate/renovate`) and GitHub container registry (`ghcr.io/renovatebot/renovate`).
 These Docker images work on all the hosting platforms that Renovate supports.
 
-You must run the Docker images in a `linux/amd64` container.
-We're working on `linux/arm64` container support.
+Both `linux/amd64` and `linux/arm64` architectures are supported, although you may still find some bugs in the `arm64` image.
 You can't run the Docker images in a Windows or macOS container.
 
 In general, you can run Renovate natively on Windows as long as you have all tools it will need (e.g. `npm`, `pipenv`, etc.) preinstalled before you run Renovate.
 
-There are two flavors:
+There are two Docker image flavors:
 
-- The `-slim` image
-- The default "full" image, this is the image you'll get if you use the `latest` tag
+- The default image, which installs required tools at runtime (default for `latest` tag),
+- The `-full` image, which comes with latest or very recent versions of every tool pre-installed
 
-##### The slim image
+##### The default image (formerly `slim`)
```

Comment:
Your changing the headings in this document, did you check if we have any links that need to be updated?